src/Fantomas/Trivia.fs

```diff
@@ -325,7 +329,7 @@ let private addTriviaToTriviaNode (startOfSourceCode:int) (triviaNodes: TriviaNo
         |> updateTriviaNode (fun tn -> { tn with ContentItself = Some siNode }) triviaNodes
 
     | { Item = Number(_) as number; Range = range  } ->
-        findNodeOnLineAndColumn triviaNodes range.StartLine range.StartColumn
+        findFirstNodeOnLineAndColumn triviaNodes range.StartLine range.StartColumn
```

Comment:
I added the `findFirstNodeOnLineAndColumn ` function as searching backwards (which `findNodeOnLineAndColumn` does) will result in the match finding the `minus` token instead of the `const`, which both have the same start column but the minus token is the latter element in the list. I can fix this in other ways too, e.g. by filtering out the `minus` token before trying to search for the node. Let me know how you'd like to have this implemented.

Reply:
I would filtered out the minus token instead.
In the of numbers we actually know that we are always looking for a SynExpr.Const node type.
So I'd go for that.

Reply:
Okay! Will do that.

Reply:
I've gone with the approach of adding a `findConstNodeOnLineAndColumn` function, which is in line with having existing `findNodeOnLineAndColumn` and `findConstNodeAfter` functions. Is this okay, or would you rather do a `List.filter` within the `| { Item = Number(_) as number; Range = range  } ->` match?